tests/test_numerapi.py

```diff
@@ -36,6 +38,36 @@ def test_download_current_dataset(api, tmpdir):
         assert len(rsps.calls) == 0
 
 
+def test_get_latest_data_paths(api):
```

Comment:
you could convert that to a pytest parametrized test, like this one: https://github.com/uuazed/numerapi/blob/9690012a8753f82373703e7041959c0d7adac31e/tests/test_numerapi.py#L49. But it's also fine like this.